<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>942</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>336</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Tank Wars&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>335</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>106</y>
      <width>952</width>
      <height>624</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="leUserName">
          <property name="placeholderText">
           <string>Name Here</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbLoad">
          <property name="text">
           <string>Start Game</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Pause">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>How to Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbEasy">
          <property name="text">
           <string>Easy</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbMedium">
          <property name="text">
           <string>Medium</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbHard">
          <property name="text">
           <string>Hard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="levels">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Levels</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="3">
            <widget class="QPushButton" name="lvl09">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="lvl08">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="lvl07">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="lvl01">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="lvl04">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="lvl06">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="lvl05">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="lvl02">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>2</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="lvl03">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="gameArea" native="true">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>600</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
